external-builds/pytorch/build_pytorch_windows.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Windows equivalent to the Linux build_pytorch_torch.sh.
+# TODO: merge into `ptbuild.py develop` or `ptbuild.py bdist_wheel` somehow?
+
+set -eo pipefail
+
+SCRIPT_DIR="$(cd $(dirname $0) && pwd)"
+
+if [ ! -n "${ROCM_HOME}" ]; then
+    export ROCM_HOME=$(realpath $SCRIPT_DIR/../../build/dist/rocm)
+    echo "ROCM_HOME: $ROCM_HOME"
+fi
+if [ -d ${ROCM_HOME} ]; then
+    export PATH=${ROCM_HOME}/bin:$PATH
+else
+    echo "Could not find ROCM_HOME: $ROCM_HOME"
+    exit 1
+fi
+
+BUILD_DIR_ROOT=${ROCM_HOME?}/../..
+
+# Environment variables recommended here:
+# https://github.com/ROCm/TheRock/discussions/409#discussioncomment-13032345
+export USE_ROCM=ON
+export USE_KINETO=0
+export BUILD_TEST=0
+export USE_FLASH_ATTENTION=0
+export USE_MEM_EFF_ATTENTION=0
+export CMAKE_PREFIX_PATH="${ROCM_HOME?}"
+# TODO(#410): Fix HIP_CLANG_PATH setting so hipcc finds the tools on its own
+export HIP_CLANG_PATH="${ROCM_HOME?}/lib/llvm/bin"
+export CC=${ROCM_HOME?}/lib/llvm/bin/clang-cl
+export CXX=${ROCM_HOME?}/lib/llvm/bin/clang-cl
+export DISTUTILS_USE_SDK=1
+
+# Match this with `-DTHEROCK_AMDGPU_FAMILIES` used to bulid TheRock.
+# TODO: pull from environment variable, option, etc.?
+export PYTORCH_ROCM_ARCH=gfx1100
```

Comment:
I wasn't expecting the GPU ISA to be hard coded here since analogous scripts like [pytorch_dev_ubuntu_24.04.Dockerfile](https://github.com/ROCm/TheRock/blob/main/dockerfiles/pytorch-dev/pytorch_dev_ubuntu_24.04.Dockerfile#L7) kept it as a build argument.

If we're transitioning this script to Python and adding aotriton support soon  then we can address how we specify the GPU ISA then.

Reply:
I'll at least convert to an argument and drop the "edit the script before you run it" from the docs. While the build steps are still up in the air, it's easier to stay in Bash. Will definitely want to move to Python soon though.

Reply:
Moved to a script argument (or used the existing env var).

Reply:
LGTM.